internal/cmd/login.go

```diff
@@ -622,13 +622,17 @@ func promptVerifyTLSCert(cli *CLI, cert *x509.Certificate) error {
 	formatTime := func(t time.Time) string {
 		return fmt.Sprintf("%v (%v)", HumanTime(t, "none"), t.Format(time.RFC1123))
 	}
+	title := "Certificate"
+	if cert.IsCA {
+		title = "Certificate Authority"
```

Comment:
I pushed this one more small change. To indicate in the prompt if the user is trusting a leaf cert or a CA.